sdk/cosmos/azure-spring-data-cosmos/README.md

```diff
@@ -283,6 +283,7 @@ Extends CosmosRepository interface, which provides Spring Data repository suppor
 @Repository
 public interface UserRepository extends CosmosRepository<User, String> {
     Iterable<User> findByFirstName(String firstName);
+    long countByFirstName(String firstName);
```

Comment:
@Blackbaud-MikeLueders - Since you added this line in readme, this should exactly match `UserRepository` class including the line numbers as well.